Gemfile

```diff
@@ -5,8 +5,7 @@ gem 'omnibus', github: 'opscode/omnibus'
 
 # Use Chef's software definitions. It is recommended that you write your own
 # software definitions, but you can clone/fork Chef's to get you started.
-#gem 'omnibus-software', github: 'opscode/omnibus-software'
-gem 'omnibus-software', github: 'opscode/omnibus-software'
+gem 'omnibus-software', github: 'opscode/omnibus-software', branch: 'irving/update_nginx'
```

Comment:
This can be changed back, since I think this branch has been merged. Also you might as well change it to:

```ruby
gem "omnibus-software", :git => "https://github.com/chef/omnibus-software.git"
```